addons/project/views/project_task_views.xml

```diff
@@ -287,7 +287,7 @@
                                 <span class="o_stat_text">in Recurrence</span>
                             </div>
                         </button>
-                        <button name="%(project_task_action_sub_task)d" type="action" class="oe_stat_button" icon="fa-tasks"
+                        <button name="project_task_action_sub_task" type="object" class="oe_stat_button" icon="fa-tasks"
```

Comment:
Why not just adding the `default_project_id` in the context here instead of creating a new action for that? 🤔